apps/daffio/src/app/content/docs/api-docs/models/api-doc-reference-list.ts

```diff
@@ -0,0 +1,11 @@
+import { DaffioApiDocReference } from './api-doc-reference';
+
+/**
+ * A list of references for API documents.
+ */
+export interface DaffioApiDocReferenceList {
+  name: string;
+  title: string;
+  path: string;
```

Comment:
What is path in this case?

Reply:
It is the path that shows in the url from `routerLink="{{doc.path}}"`. Then that path is taken by the api-doc/service and used as a directory lookup to retrieve the actual document.